src/configuration.ts

```diff
@@ -866,6 +904,28 @@ export function getBuildLogForConfiguration(configuration: string | undefined):
     }
 }
 
+export function getPreConfigureArgsForConfiguration(configuration: string | undefined): void {
+    let makefileConfiguration: MakefileConfiguration | undefined = getMakefileConfiguration(configuration);
+    const preConfigArgs = makefileConfiguration?.preConfigureArgs;
```

Comment:
Too late (or not the right moment) to rename preConfigureArgs into globalPreConfigureArgs.... to be obviously different than an eventual localPreConfigureArgs, so if we leave preConfigureArgs alone (and we know those are the global ones) then let's rename preConfigArgs into localPreConfigureArgs. Otherwise, Config versus Configure in the two names is a not clear what is the difference and we'd have to read the code again to understand which is what.

Reply:
👍 Done.